packages/common/test/data/paramsTest.ts

```diff
@@ -0,0 +1,52 @@
+import type { ParamsDict } from '@ethereumjs/common'
+
+export const paramsTest: ParamsDict = {
+  /**
+   * Frontier/Chainstart
+   */
+  1: {
+    // pow
+    minerReward: BigInt('5000000000000000000'), // the amount a miner get rewarded for mining a block
+  },
+  /**
+   * Byzantium HF Meta EIP
+   */
+  609: {
+    // gasPrices
+    ecAddGas: 500, // Gas costs for curve addition precompile
+    // pow
+    minerReward: BigInt('3000000000000000000'), // the amount a miner get rewarded for mining a block
+  },
+  /**
+.  * Constantinope HF Meta EIP
+.  */
+  1013: {
+    // gasPrices
+    netSstoreNoopGas: 200, // Once per SSTORE operation if the value doesn't change
+    // pow
+    minerReward: BigInt('2000000000000000000'), // The amount a miner gets rewarded for mining a block
+  },
+
```

Comment:
```suggestion
```
A bit of a nit but maybe spacing can be kept consistent.

Reply:
Updated